packages/db/src/lib/granule.ts

```diff
@@ -354,3 +354,58 @@ export const getGranulesByGranuleId = async (
     .where({ granule_id: granuleId });
   return records;
 };
+
+/**
+ * Change a granules' PG record and its files' PG record based on collection move
+ * 
+ * @param {Knex | Knex.Transaction} knexOrTransaction - DB client or transaction
+ * @param {Object} [collectionPgModel] - Collection PG model class instance
+ * @param {string[]} granuleIds - list of granules by granuleIds to change
+ * @param {string} collectionId - collection ID
+ * @returns {Promise<void>}
+ */
+ export const updateGranuleAndFiles = async (
+  knexOrTransaction: Knex | Knex.Transaction,
+  collectionPgModel = new CollectionPgModel(),
```

Comment:
can this be removed from params or moved to the last param? I think the default is going to be to let it be a new collectionPgModel